backend/Pipfile

```
@@ -58,4 +59,4 @@ flake8-quotes = "*"
 requests = "*"
 
 [requires]
-python_version = "3.11.0" # from .python-version
+python_version = "3.11.2"
```

.python-version er source of truth. Den må oppdateres også.
```suggestion
python_version = "3.11.2" # from .python-version
```

---

Fix